execution/execution_test.go

```diff
@@ -1228,11 +1232,14 @@ func TestContractSend(t *testing.T) {
 	}
 
 	exe := makeExecutor(st)
+	fmt.Println("acc0", acc0.Address)
```

Comment:
Let's test these for not nil

Reply:
I will fix this, though these are just stray debug lines.

Alas there are quite a few unchecked errors in this test. Most of which are guaranteed not to be errors in the context of these tests but still that should be made explicit. execution_test needs a complete rework. I will open a ticket for this.